### PR TITLE
feat(frontend): simple token group store

### PR DESCRIPTION
**Pull request by darioloetsc**
# Motivation

Introduce a token group store to preserve expanded settings in the grouping view.

# Changes
Add token group store.

# Tests

### CODE REVIEW

**Review by AntonioVentilii-DFINITY — COMMENTED** [review]
I think there was a misunderstanding: your first iteration was ok, I would have just used the `initCertifiedSetterStore`

So, that the definition would have been

```typescript
export interface TokenGroupData {
	expanded: boolean;
}

//todo: replace with TokenGroupId
export const tokenGroupStore = initCertifiedSetterStore<TokenGroupData>();
```

**Comment by darioloetsc** [comment]
> I think there was a misunderstanding: your first iteration was ok, I would have just used the `initCertifiedSetterStore`
> 
> So, that the definition would have been
> 
> ```ts
> export interface TokenGroupData {
> 	expanded: boolean;
> }
> 
> //todo: replace with TokenGroupId
> export const tokenGroupStore = initCertifiedSetterStore<TokenGroupData>();
> ```

I definitely misunderstood that one, here we go eff256bc.

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM! TKs

File: src/frontend/src/lib/stores/token-group.store.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { initCertifiedSetterStore } from '$lib/stores/certified-setter.store';
+
+export interface TokenGroupData {
+	expanded: boolean;
+}
+
+export const tokenGroupStore = initCertifiedSetterStore<TokenGroupData>();
```
Comment (peterpeterparker) [inline src/frontend/src/lib/stores/token-group.store.ts:7]: @AntonioVentilii-DFINITY but those kind of data are never certified? or is there plan to save the information in the backend?

**Comment by peterpeterparker** [comment]
Maybe I should have blocked this - i.e. as mentionned [above](https://github.com/dfinity/oisy-wallet/pull/3070#discussion_r1812872179), why are we using a certified store?

**Pull request merged**